src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx

```diff
@@ -261,6 +303,28 @@ The configuration file has common settings applicable to all integrations like `
 
   </Collapser>
 
+  <Collapser
+    id="basic-config-gmsa-domain"
+    title="Basic configuration with Windows Group Managed Service Account"
+  >
+    This is the basic configuration used to collect metrics and inventory from your localhost. In this case we use a Group Managed Service Acccount for authentication which requires a blank username and password:
```

Comment:
I see we're using gMSA interchangeable with Group Managed Service Account. Is gMSA just the acronym for this account type?

Reply:
gMSA is just the acronym for a Group Managed Service account. This is a Microsoft concept that can be used in Windows and SQL Server.